backend/routes/indexer/combineTokenId.go

```diff
@@ -0,0 +1,18 @@
+package indexer
+
+import (
+	"math/big"
+)
+
+
+func combineLowAndHighTokenId(tokenIdLowInt, tokenIdHighInt int64) (*big.Int) {
+	// Convert int64 parts to big.Int
+	low :=  new(big.Int).SetInt64(tokenIdLowInt)
```

Comment:
If we parse low and high as int64, there will be 64 bits of data chopped off of each of the low and high value on the LHS.

Each low and high are encoded as left padded u128. I think the best way to create this function would be to take the low and high as their hex string, then properly combine the strings while removing the left padding 0s. Then you can interpret the string as u256 ( big.Int should have a from string like function )

Reply:
Noted.